core/modules/util/common.h

```diff
@@ -93,6 +94,81 @@ std::ostream& printList(std::ostream &os, char const* label, C const& c) {
     return os;
 }
 
+template <typename C>
```

Comment:
I'm not sure this is the best place for them. qserv::global::debugUtil.h also looks like another reasonable place to put them. Any thoughts?